src/definitions/modules/dropdown.less

```diff
@@ -565,13 +565,23 @@ select.ui.dropdown {
     padding: @selectionItemPadding;
     white-space: normal;
     word-wrap: normal;
+    z-index: 0;
   }
 
   /* User Item */
   .ui.selection.dropdown .menu > .hidden.addition.item {
     display: none;
   }
 
+  /* User addition item */
+  .ui.dropdown > .menu > .addition.item:first-child {
```

Comment:
we may make this feature optional anyway by using an necessary extra classname like "sticky" (save to use as long as "ui" is not also used inside the item, otherwise let's use "stuck" instead
```css
.ui.dropdown > .menu > .sticky.addition.item:first-child {
or
.ui.dropdown > .menu > .stuck.addition.item:first-child {
```

Reply:
alright, thx for the review!
I defined a new variable, `@additionZIndex`, which is basically `@hoveredZIndex + 1`. We don't need to override hover z-index but we need the addition z-index to be gt than it. And I used it :)